Add post: communicating ENIC to international candidates (#1304)
DFE-Digital/bat-design-history: app/posts/apply-for-teacher-training/2024-09-02-communicating-enic-to-international-candidates.md

---
title: Helping international candidates show that their qualifications are equivalent to UK ones
description: Communicating the certificate of comparability to international candidates so that they can apply for one at the right time in their application.
date: 2024-09-02
tags:
    - ENIC
    - international
    - qualifications

> 

Reviewer: ```suggestion
```

---

## The problem we are trying to solve

In the 2023 to 2024 recruitment cycle we saw a 269% increase in applications from international candidates compared to the previous cycle, but a 25.8% reduction in the success rate of these applications.

Training providers reported that they were struggling to process the high number of applications because of the extra time required to check qualifications gained overseas to make sure that they match the entry requirements.

>  It can take providers 10 times longer to process applications from international candidates. (From user research in December 2023)

Reviewer: ```suggestion
It can take providers 10 times longer to process applications from international candidates (From user research in December 2023).
```

> Applicants are able to buy a certificate known as a statement of comparability from UK National Information Centre (UK ENIC) which shows how their international qualifications compare to ones gained in the UK.

Reviewer: ```suggestion
Applicants can buy a certificate, known as a statement of comparability, from the [UK National Information Centre (UK ENIC)](https://www.enic.org.uk/), which shows how their international qualifications compare to ones gained in the UK.
```

The statement of comparability makes it easier for training providers to process applications from overseas because they can see how their qualifications compare to UK ones. It also increases the likelihood of the applicant receiving an offer by 28% (in the 2023 to 2024 cycle).

We wanted to increase the number of applications who submit with a statement of comparability to reduce the amount of time it takes providers to process international applications and increase the number of successful applications from international candidates.

## Research and analysis

(From the International Research review in February 2024)

Candidates said they:

> * don't find out they need an ENIC until quite late in the application process
> * don’t know what ENIC is
> * don’t know why ENIC is important
> * don’t know that it takes 15 working days to receive your ENIC reference number

Reviewer: Try to avoid negative contractions.

```suggestion
* do not find out they need an ENIC until quite late in the application process
* do not know what ENIC is
* do not know why ENIC is important
* do not know that it takes 15 working days to receive your ENIC reference number
```
